homeassistant/components/minecraft_server/binary_sensor.py

```diff
@@ -30,16 +48,17 @@ async def async_setup_entry(
 class MinecraftServerStatusBinarySensor(MinecraftServerEntity, BinarySensorEntity):
     """Representation of a Minecraft Server status binary sensor."""
```

Comment:
```suggestion
class MinecraftServerBinarySensor(MinecraftServerEntity, BinarySensorEntity):
    """Representation of a Minecraft Server binary sensor."""
```

Reply:
Oops, good catch, thanks 👍 
I'll rename it to `MinecraftServerBinarySensorEntity`, to be consistent with the sensor platform.

Reply:
Sure!